crates/sui-move/src/build.rs

```diff
@@ -41,6 +41,9 @@ impl Build {
         build_config: MoveBuildConfig,
     ) -> anyhow::Result<()> {
         let rerooted_path = base::reroot_path(path)?;
+        let lock_file_path = rerooted_path.join("Move.lock");
```

Comment:
Given this crate uses this functionality too, should `resolve_lock_file_path` belong here?

Reply:
Agreed that `sui-move` importing from `client_commands` is strange -- I'm suggesting the other way around -- move the helper into `sui-move` and import it into `client_commands`.